admin_manual/configuration_server/occ_command.rst

```diff
@@ -555,15 +555,77 @@ authentication servers such as LDAP::
 Upgrade Command
 ---------------
 
+List all options, like this example on CentOS Linux::
+
+ $ sudo -u apache php occ upgrade -h
+ Usage:
+ upgrade [--skip-migration-test] [--dry-run] [--no-app-disable]
+
+ Options:
+ --skip-migration-test  skips the database schema migration simulation and 
+    update directly
+ --dry-run              only runs the database schema migration simulation, do 
+   not actually update
+ --no-app-disable       skips the disable of third party apps
+ --help (-h)            Display this help message.
+ --quiet (-q)           Do not output any message.
+ --verbose (-v|vv|vvv)  Increase the verbosity of messages: 1 for normal output, 
+   2 for more verbose output and 3 for debug.
+ --version (-V)         Display this application version.
+ --ansi                 Force ANSI output.
+ --no-ansi              Disable ANSI output.
+ --no-interaction (-n)  Do not ask any interactive question
+
 When you are performing an update or upgrade on your ownCloud server (see the 
 Maintenance section of this manual), it is better to use ``occ`` to perform the 
 database upgrade step, rather than the Web GUI, in order to avoid timeouts. PHP
 scripts invoked from the Web interface are limited to 3600 seconds. In larger 
 environments this may not be enough, leaving the system in an inconsistent 
 state. After performing all the preliminary steps (see 
-:doc:`../maintenance/upgrade`) use this command to upgrade your databases::
+:doc:`../maintenance/upgrade`) use this command to upgrade your databases, 
+like this example on CentOS Linux. Note how it details the steps::
 
  $ sudo -u www-data php occ upgrade
+ ownCloud or one of the apps require upgrade - only a limited number of 
+ commands are available                            
+ Turned on maintenance mode                                                      
+ Checked database schema update           
+ Checked database schema update for apps
+ Updated database      
+ Updating <gallery> ...                                                          
+ Updated <gallery> to 0.6.1               
+ Updating <activity> ...
+ Updated <activity> to 2.1.0            
+ Update successful
+ Turned off maintenance mode
+ 
+If there is an error it throws an exception, and the error is detailed in your 
+ownCloud logfile, so you can use the log output to figure out what went wrong, 
+or to use in a bug report::
+
+ Turned on maintenance mode
+ Checked database schema update
+ Checked database schema update for apps
+ Updated database
+ Updating <files_sharing> ...
+ Exception
```

Comment:
It's not a real live exception, but should illustrate that you then better know what happened ;)